cfme/tests/intelligence/test_reports_with_timelines.py

```diff
@@ -9,8 +9,8 @@
 def test_default_reports_with_timelines(report):
     """
     Polarion:
-        assignee: None
-        initialEstimate: None
+        assignee: jdupuy
+        initialEstimate: 1/5h
     """
```

Comment:
Please add following tags to polarion information:
- steps for this test case under ``` testSteps: ```
- also add ```caseimportnace``` and ```casecomponent```

Reply:
@ganeshhubale I added some more Polarion data in my most recent push.